src/originalBudgetTrackingApp/import/importTransactions.ts

```diff
@@ -1,48 +1,57 @@
 import moment from 'moment';
 import _ from 'lodash';
-import { Config } from '@/originalBudgetTrackingApp/configManager/configManager';
+import * as configManager from '@/originalBudgetTrackingApp/configManager/configManager';
 import { EnrichedTransaction } from '@/originalBudgetTrackingApp/commonTypes';
 import * as bankScraper from '@/originalBudgetTrackingApp/import/bankScraper';
-import * as configManager from '@/originalBudgetTrackingApp/configManager/configManager';
 import { ScaperScrapingResult, Transaction } from '@/originalBudgetTrackingApp/import/bankScraper';
 import * as categoryCalculation from '@/originalBudgetTrackingApp/import/categoryCalculationScript';
+import { BudgetTrackingEventEmitter, EventNames } from '../eventEmitters/EventEmitter';
 
 type AccountToScrapeConfig = configManager.AccountToScrapeConfig;
+type Config = configManager.Config;
 type ScrapingConfig = Config['scraping'];
 
 const TRANSACTION_STATUS_COMPLETED = 'completed';
-const DATE_FORMAT = 'DD/MM/YYYY';
 
-export async function scrapeFinancialAccountsAndFetchTransactions(scrapingConfig: ScrapingConfig, startDate: Date) {
+export async function scrapeFinancialAccountsAndFetchTransactions(scrapingConfig: ScrapingConfig, startDate: Date, eventEmitter: BudgetTrackingEventEmitter) {
```

Comment:
```suggestion
export async function scrapeFinancialAccountsAndFetchTransactions(scrapingConfig: ScrapingConfig, startDate: Date, eventEmitter?: BudgetTrackingEventEmitter) {
```
Should be optional I think, or default one, if we don't want to handle null (although it is just to add `?` in `eventEmiter?.emit`

Reply:
Sorry, ignore this. I see the `scrapeAndUpdateOutputVendors` function